packages/traceability-schemas/scripts/build-test-vectors.js

```diff
@@ -101,10 +103,54 @@ Object.keys(schemas).forEach((schemaName) => {
       }
       const good = JSON.parse(fs.readFileSync(exampleFile));
       let credTemplate = require('../src/data/vc/vc.json');
+      const Evidence = require('../src/data/vc/evidence.json');
+      const Credentialstatus = require('../src/data/vc/credentialstatus.json');
+      delete Evidence['@context'];
+      // detect if evidence property is to be included
```

Comment:
nit: these 2 code block would be much more readable as functions.

Reply:
@or13 ... done! thanks for the nitting ... makes the code better rather than just me hacking ;-)